easybuild/easyconfigs/s/SMARTdenovo/SMARTdenovo-20180219-foss-2018b.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = "ConfigureMake"
+
+name = 'SMARTdenovo'
+version = '20180219'
+
+homepage = 'https://github.com/ruanjue/smartdenovo'
+description = """SMARTdenovo is a de novo assembler for PacBio and
+Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw
+read alignments without an error correction stage. It also provides
+tools to generate accurate consensus sequences, though a platform
+dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish
+for ONT) are still required for higher accuracy."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+commit = '5cc1356bffb87197b0fb9b835cfe1adc5a764dc6'
+sources = [{
+    'filename': '%%(name)s-master-20190215-%s.tar.gz' % commit,
+    'git_config': {
```

Comment:
@akesandgren When the sources live on GitHub you don't need to use `git` at all, you can easily get a tarball for a specific commit (with a fixed checksum)

Done in https://github.com/akesandgren/easybuild-easyconfigs/pull/9